debian/rules

```diff
@@ -15,9 +15,16 @@ ifneq (,$(filter parallel=%,$(DEB_BUILD_OPTIONS)))
     MAKEFLAGS += -j$(NUMJOBS)
 endif
 
+ifneq (,$(filter BUILD_TYPE=%,$(SCONS_ARGS)))
+    BUILD_TYPE = $(patsubst build_type=%,%,$(filter BUILD_TYPE=%,$(SCONS_ARGS)))
+else
+    BUILD_TYPE = release
```

Comment:
This defeats our "secure by default" stance.  IOW, this should be the case without even having to use `BUILD_TYPE=release` and then we should have a test that confirms it.

Reply:
The above code has been removed and the SCONS_ARG will be added to the DEB_SCONS_ARGS.  If no BUILD_TYPE is specified it will use the default.   The default for BUILD_TYPE is determined in base.py and is currently defaulted to "dev".  I will open a separate ticket as to what the default of BUILD_TYPE should be.